frontend/talentsearch/src/js/components/request/CreateRequest.tsx

```diff
@@ -52,7 +53,7 @@ type FormValues = {
       sync?: Array<Maybe<Skill["id"]>>;
     };
     hasDiploma?: ApplicantFilterInput["hasDiploma"];
-    wouldAcceptTemporary?: ApplicantFilterInput["wouldAcceptTemporary"];
+    positionDuration?: ApplicantFilterInput["positionDuration"];
```

Comment:
I noticed that the request page still shows "Indeterminate duration (permanent)" if you don't select anything on the search page.  Since we don't want to treat this as boolean any longer I think it would make more sense if it showed the "(None selected)" string.  And left that field as undefined in the mutation.
![image](https://user-images.githubusercontent.com/8978655/204839007-f75e8a22-ae5c-4b26-8308-0651d7e18284.png)
Similarily, I think the permanent radio should send "PERMANENT" and not null to the mutation.